pkg/api/resource.go

```diff
@@ -24,24 +24,30 @@ type Resource struct {
 
 type CreateMethod struct {
 	SupportsUserSettableCreate bool
+	Parameters                 []openapi.Parameter
```

Comment:
once you add this, there's a few other places where this needs to be wired through:

1. The parameters when re-serializing this to openapi: https://github.com/aep-dev/aep-lib-go/blob/main/pkg/api/openapi.go#L95.
2. Writing of the proto in aepc: https://github.com/aep-dev/aepc/blob/main/writer/proto/resource.go#L45 (the lockstep change that's required probably implies we need to move the proto code upstream).